agent/src/agent/pipeline/config/stages/js_convert_metrics.py

```diff
@@ -5,9 +5,9 @@ class JSConvertMetrics(base.Stage):
     def get_js_vars(self):
         return f"""
 state['TIMESTAMP_COLUMN'] = '{self.pipeline.timestamp_path}';
-state['DIMENSIONS'] = {self.pipeline.dimensions_paths};
-state['DIMENSIONS_NAMES'] = {self.pipeline.dimensions_names};
-state['VALUES_COLUMNS'] = {self.pipeline.values_paths};
+state['DIMENSIONS'] = {self.pipeline.dimension_paths};
```

Comment:
why not plural? it sounds like it's multiple paths of a single dimension

Reply:
in English it's correct to say `dimension paths, I checked it with Brian